src/stories/index.tsx

```diff
@@ -0,0 +1,13 @@
+import * as S from './styles'
```

Comment:
Algum motivo especial para seguir esse padrao de colocar os objetos criados de estilo em `S`?
Ja vi muita gente utilizando, eu particularmente nao curto muito, acho que suja um pouco o codigo.

Reply:
À medida que estava aprendendo sobre styled-components, vi essa prática sendo adotada pra diferenciar o que é tag de estilização e o que é componente, daí achei bacana usar! Acabei acostumando um pouco.

Reply:
Se voce se sente confortavel com essa pratica pode continuar, se for continuar. Nao tem um certo e errado, o que existe e um guide line, tem empresas que utilizando da maneira que vc fez, outras nao, e sempre bom seguir o padrao que a empresa utiliza, nesse caso que o projeto e seu voce tem a liberdade de escolher, mas se escolher de uma maneira todos os outros componentes tem que seguir.